docs/reference/configuration.md

```diff
@@ -0,0 +1,114 @@
+## Configuration Reference
+
+<img src="../../assets/structure.svg" style="background: #eff1f5" />
```

Comment:
The CSS in the `style` tag seems to be getting stripped out by GitHub:

![image](https://github.com/user-attachments/assets/6092504f-fa10-45c8-b21a-db1a80208658)

Probably best to convert to a png?